lib/dependencies/ImportMetaContextDependencyParserPlugin.js

```diff
@@ -200,6 +200,27 @@ module.exports = class ImportMetaContextDependencyParserPlugin {
 								}
 								break;
 							}
+							case "fetchPriority": {
+								const expr = parser.evaluateExpression(
+									/** @type {ExpressionNode} */ (prop.value)
+								);
+								if (expr.isBoolean()) {
+									groupOptions.fetchPriority = "auto";
```

Comment:
@aboktor , what is the purpose of allowing a boolean at all, when setting it (and regardless of the value) will result in `"auto"`. I would think just keeping it to the three enums -- "high", "low", "auto" -- would make more sense